pkg/persistence/postgrespersister.go

```diff
@@ -171,62 +177,47 @@ func (p *PostgresPersister) retrieveDbEvents(tableName string, criteria *model.R
 	if criteria.Count > 0 {
 		queryBuf.WriteString(" LIMIT :count") // nolint: gosec
 	}
-
-	dbEvent := []postgres.Event{}
-	nstmt, err := p.db.PrepareNamed(queryBuf.String())
-	if err != nil {
-		return nil, err
-	}
-	err = nstmt.Select(&dbEvent, criteria)
-	return dbEvent, err
+	return queryBuf.String()
 }
 
-// PopulateBlockDataFromDB will fill the persistence with data from the DB.
-func (p *PostgresPersister) PopulateBlockDataFromDB(tableName string) error {
-	events, err := p.getLatestEvents(tableName)
+// actually can make this even more generic and put in utils
```

Comment:
Minor, but is this a NOTE or TODO?

Reply:
oh ooops, this is another comment to myself i left in here. decided against it so will remove